services/director-v2/tests/unit/test_utils_osparc_variables.py

```diff
@@ -137,18 +138,20 @@ def mock_user_repo(mocker: MockerFixture, mock_repo_db_engine: None) -> None:
 
 
 @pytest.fixture
-async def variables_app() -> AsyncIterable[FastAPI]:
+async def fake_app(faker: Faker) -> AsyncIterable[FastAPI]:
     app = FastAPI()
     app.state.engine = AsyncMock()
 
+    app.state.settings = SimpleNamespace(DIRECTOR_V2_PUBLIC_API_BASE_URL=faker.url())
```

Comment:
interesting trick.. so you bypass the settings setup with this hack.

Reply:
the fixture was already there. In this context, it makes sense not to create the entire app. so I could have also done it with a mock, as with the engine. Probably it is even better